js/src/elements/form-behavior.ts

```diff
@@ -686,12 +689,12 @@ export function navigateAfterSubmit(href: string | null, navigationStep: Navigat
 	switch (navigationStep) {
 		case null:
 			// do nothing.
-			break;
+			return false;
 		case 'follow':
 			if (href) {
 				location.href = href;
 			}
-			break;
+			return true;
```

Comment:
does this need to return false when there's no href?

```
if (href) {
  location.href = href;
  return true;
}
return false;
```

Reply:
Updated! I thought this would always reload the page with or without an href, but have learned otherwise